my-app/src/pages/authentication/Login.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {useContext} from 'react';
+import styled from "styled-components";
+import {Link} from "react-router-dom";
+import {Button, Paper, TextField, Typography} from "@mui/material";
+import {LoginContext} from "./AuthContext";
+
+
+export const Login = () => {
+    const logic = useContext(LoginContext)
+
+return (
+
+    <PaperWrapper>
+

```

Comment:
I don't see how you use prettier, it seems like you don't format your code. 

try to 
- move `eslinttrc.json` and `.prettierrc.js` from `src` to `my-app`, it should be on the same level with package json
- add "format" script to your scripts in package.json `"format": "prettier --config .prettierrc --write \"src/**/*.ts?(x)\"",`
- rename `.prettierrc.js` to `.prettierrc` and change its contents to 
```
{
  "printWidth": 80,
  "tabWidth": 2,
  "useTabs": false,
  "semi": false,
  "singleQuote": true,
  "quoteProps": "as-needed",
  "jsxSingleQuote": true,
  "trailingComma": "none",
  "bracketSpacing": true,
  "jsxBracketSameLine": false,
  "arrowParens": "always",
  "endOfLine": "lf"
}
```

Reply:
then run `yarn format` or `npm format` in your terminal.

Reply:
fixed